static/app/components/profiling/flamegraphZoomView.tsx

```diff
@@ -326,9 +325,8 @@ function FlamegraphZoomView({
       setConfigSpaceCursor(null);
     };
 
-    const onZoomIntoFrame = (frame: FlamegraphFrame) => {
+    const onZoomIntoFrame = () => {
       setConfigSpaceCursor(null);
-      dispatchFlamegraphState({type: 'set selected node', payload: frame});
```

Comment:
Doesn't this break the zoom behaviour?

Reply:
Just checked again, looks good. We already setSelected node when a user clicks, this looks like it was  redundant before. It also implicitly tied the zoom to the selected node, which I want to get rid of (so that we can zoom and move into nodes) without marking them selected if we need to
![CleanShot 2022-05-31 at 14 28 24](https://user-images.githubusercontent.com/9317857/171258988-2396e889-6be2-4b31-99dc-61e6747bb737.gif)